dcos-log/api/v2/handlers.go

```diff
@@ -637,7 +637,7 @@ func browseFiles(w http.ResponseWriter, req *http.Request) {
 	}
 
 	if err := json.NewEncoder(w).Encode(files); err != nil {
-		logError(w, req, fmt.Sprintf("unable to encode sandbox files: %s. Items: %s", err, files), http.StatusInternalServerError)
+		logError(w, req, fmt.Sprintf("unable to encode sandbox files: %s. Items: %x", err, files), http.StatusInternalServerError)
```

Comment:
This seems unrelated to the message included in this commit message "Update dependecies". It should probably be in its own commit.

Reply:
Latest `go vet` catch this errors and build will fail.